docs/src/charm/howto/openstack.md

```diff
@@ -0,0 +1,120 @@
+# Integrating with OpenStack
+
+This guide explains how to integrate {{product}} with the OpenStack cloud
+platform.
+
+## Prerequisites
+
+To follow this guide, you will need:
+
+- An [OpenStack][openstack] cloud environment.
+- Juju set-up as the deployment tool for Openstack. Please refer to
```

Comment:
```suggestion
- Juju set-up as the deployment tool for OpenStack. Please refer to
```
:stuck_out_tongue:

Reply:
Look at this guy... epic!  Next thing you know, he'll want us spelling tires with a y